frontend/src/routes/Governance/policies/policy-details/PolicyDetailsOverview.test.tsx

```diff
@@ -50,6 +60,35 @@ describe('Policy Details Results', () => {
     await waitForText('Without violations:')
   })
 
+  test.each([
+    [[policy2, policy2], REMEDIATION_ACTION.ENFORCE_OVERRIDDEN],
+    [[policy2, policy3], REMEDIATION_ACTION.INFORM_ENFORCE_OVERRIDDEN],
+    [[policy3, policy2], REMEDIATION_ACTION.INFORM_ENFORCE_OVERRIDDEN],
+    [[policy3, policy3], REMEDIATION_ACTION.INFORM],
+  ])(
+    'Should render Policy Details Results Page correctly for policy override',
+    async (propagatedPolicies, expected) => {
+      render(
+        <RecoilRoot
+          initializeState={(snapshot) => {
+            snapshot.set(placementsState, mockPlacements)
+            snapshot.set(policySetsState, [mockPolicySets[0]])
+            snapshot.set(placementBindingsState, mockPlacementBindings)
+            snapshot.set(placementDecisionsState, mockPlacementDecision)
+            snapshot.set(policiesState, propagatedPolicies)
+          }}
+        >
+          <MemoryRouter>
+            <PolicyDetailsOverview policy={mockPolicy[0]} />
+          </MemoryRouter>
+        </RecoilRoot>
+      )
+
+      // wait page load
+      await waitForText(expected)
```

Comment:
Could you please make this selector more specific? This just checks for any text on the screen.

Reply:
This only test the PolicyDetailsOverview component not the whole screen `Remediation` only shows on this component... also other tests also checking like this

Reply:
I suppose it's okay since the mock data is very controlled.